target/linux/rockchip/armv8/config-6.1

```diff
@@ -363,6 +363,7 @@ CONFIG_MDIO_BUS_MUX_GPIO=y
 CONFIG_MDIO_BUS_MUX_MMIOREG=y
 CONFIG_MDIO_DEVICE=y
 CONFIG_MDIO_DEVRES=y
+CONFIG_MEDIATEK_GE_PHY=y
```

Comment:
why? this is only used for mediatek's built-in Ge PHY.

Reply:
`CONFIG_MEDIATEK_GE_SOC_PHY` is for built-in PHYs in MediaTek MT798x SoCs, but `CONFIG_MEDIATEK_GE_PHY` is for the PHYs in MediaTek MT753x switches. This is selected by `CONFIG_NET_DSA_MT7530_MDIO`.

https://elixir.bootlin.com/linux/v6.1.85/source/drivers/net/phy/mediatek-ge.c#L71